### PR TITLE
[ICA] Manhole Cover - 3408

### PR DESCRIPTION
**Pull request by QingAn**
Manhole Cover object from ICA

### CODE REVIEW

**Comment by seanmcilroy29** [comment]
@QingAn - Could you update your Object submission with the following;

- Object version 1.0
- Object URN with - urn:oma:lwm2m:ext:3408

File: 3408.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+				<Units>rad</Units>
+				<Description><![CDATA[The tilt angle of a manhole cover]]></Description>
+			</Item>
+			<Item ID="3">
+				<Name>Cover Open Status</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration>Not detected open (0) or Detected open (1)</RangeEnumeration>
```
Comment (akeranen) [inline 3408.xml]: This field can be empty since only two values are accepted: true and false

**Review by ScottTaftPotter — COMMENTED** [review]
For Safety Angle and Manhole Cover Tilt, the units are radians, but the range enumeration is 0-180 which appears to be degrees.

File: 3408.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+				<Units></Units>
+				<Description><![CDATA[Indicate whether an open status of the manhole cover is detected]]></Description>
```
Comment (akeranen) [inline 3408.xml]: This sounds a bit convoluted. Maybe something like: "Indicates whether the manhole cover is detected open (true) or closed (false)".

File: 3408.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		<Resources>
+			<Item ID="1">
+				<Name>Safety Angle Threshold</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0―180</RangeEnumeration>
+				<Units>rad</Units>
+				<Description><![CDATA[The threshold of manhole cover's safety angle]]></Description>
```
Comment (akeranen) [inline 3408.xml]: Please add full stops at the end of descriptions.

File: 3408.xml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="1">
+				<Name>Safety Angle Threshold</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0―180</RangeEnumeration>
```
Comment (akeranen) [inline 3408.xml]: If the unit is radians, the range is probably different than 0-180? And type should be float? Or if degrees is really desirable, it's possible to register that as alternative unit in the new SenML units registry.

**Pull request merged**